docs/UserGuide.md

```diff
@@ -16,21 +16,20 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
    > Having any Java 8 version is not enough. <br>
    > This app will not work with earlier versions of Java 8.
 
-1. Download the latest `addressbook.jar` from the [releases](../../../releases) tab.
-2. Copy the file to the folder you want to use as the home folder for your Address Book.
+1. Download the latest `dueue.jar` from the [releases](../../../releases) tab.
+2. Copy the file to the folder you want to use as the home folder for your Dueue.
 3. Double-click the file to start the app. The GUI should appear in a few seconds.
    > <img src="images/Ui.png" width="600">
 
 4. Type the command in the command box and press <kbd>Enter</kbd> to execute it. <br>
    e.g. typing **`help`** and pressing <kbd>Enter</kbd> will open the help window.
 5. Some example commands you can try:
-   * **`list`** : lists all contacts
-   * **`add`**` John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01` :
-     adds a contact named `John Doe` to the Address Book.
-   * **`delete`**` 3` : deletes the 3rd contact shown in the current list
+   * **`list`** : lists all tasks
+   * **`add`**`  add laundry due/every sunday #personal d/wash clothes @B1 *f` :
+     adds a task to Dueue named `laundry` which is repeating `every sunday` under list `personal` with description `wash clothes` at venue `B1` and star it as `favourite`.
+   * **`delete`**` 3` : deletes the task with index 3 as shown in the current list
    * **`exit`** : exits the app
-6. Refer to the [Features](#features) section below for details of each command.<br>
-
+6. Refer to the [Features](#2-features) section below for details of each command.<br>
 
 ## 2. Features
```

Comment:
We have the option for user not to specify any date/time, so by doing that the user can group them in lists. And if the user does not even specify the list, it will automatically be placed in the Inbox. (which may not have been specified here)